dm/tests/shardddl4_1/run.sh

```diff
@@ -975,23 +975,32 @@ function DM_155_CASE {
 	run_sql_source2 "insert into ${shardddl1}.${tb2} values(27,27,27);"
 
 	run_sql_source2 "alter table ${shardddl1}.${tb2} change c b int;"
-	sleep 1
+	check_log_contain_with_retry "finish to handle ddls in optimistic shard mode.*alter table ${shardddl1}.${tb1} change c b int" \
```

Comment:
there's a lot of `run-sql & sleep & random-restart` pattern before and after this block, should that need to be changed?

Reply:
yes, there is a lot of `random-restart`. but the reason of the failure may be shardding ddls is not completed, update table checkpoint but not global checkpoint, and then restart to sync, it returns `Column count doesn't match value count: 4 (columns) vs 5 (values)`, so it needs to wait sharding group is completed and just this three ddls will finish sharding group and write checkpoint, other ddls will skip or create compatible table structure. so other ddls maybe   still ok without check.